### PR TITLE
chore(rxCollapse): FRMW-510 update demo with rxMetadata

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-510

- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

---
Update `rxCollapse` demo to use `rxMetadata`.

### CODE REVIEW

**Comment by 100stacks** [comment]
### Before - rxCollapse collapsed

screen shot 2015-11-27 at 10 57 14 am
 
### Before - rxCollapse expanded

screen shot 2015-11-27 at 10 57 32 am

**Comment by 100stacks** [comment]
### After - rxCollapse using rxMetadata collapsed

screen shot 2015-11-27 at 11 01 16 am
 
### After - rxCollapse using rxMetadata expanded

screen shot 2015-11-27 at 11 01 35 am

File: src/components/rxCollapse/docs/rxCollapse.html [hunk 1]
```diff
@@ -2,28 +2,20 @@
...
-    <dl>
-        <dt>Name:</dt>
-        <dd>Lorem ipsum dolor sit amet</dd>
-        <dt>ID:</dt>
-        <dd>1aa2bfa9-de8d-42f7-9f6d-e6437855b36e</dd>
-        <dt>Region:</dt>
-        <dd>ORD</dd>
-        <dt>Created:</dt>
-        <dd>December 2, 2014 @ 14:28</dd>
+<h3>Example</h3>
```
Comment (glynnis) [inline src/components/rxCollapse/docs/rxCollapse.html]: Could you make this title more descriptive? Something like:
> 'See more/See less' for use with metadata

**Comment by glynnis** [comment]
Once the "Example" title is updated, this LGTM :+1:

**Comment by 100stacks** [comment]
@glynnis recommendation incorporated :+1:

**Comment by 100stacks** [comment]
@CITguy just curious, I pushed:

 `$git push upstream FRMW-510-update-rxCollapse-demo-use-metadata` 

...to trigger visual regression on this PR.  Did I miss a step, or does it trigger on certain elements/file(CSS/LESS) changes?

**Comment by CITguy** [comment]
@100stacks, there's no visual regression set up yet for rxMetadata. This is something that will need to be set up in version 2.x of the framework (or potentially sooner).

File: src/components/rxCollapse/docs/rxCollapse.html [hunk 2]
```diff
@@ -2,28 +2,20 @@
...
         <rx-collapse class="demo-no-title" expanded="false">
-	        <dt>Name:</dt>
-	        <dd>Lorem ipsum dolor sit amet</dd>
-	        <dt>ID:</dt>
-	        <dd>1aa2bfa9-de8d-42f7-9f6d-e6437855b36e</dd>
-	        <dt>Region:</dt>
-	        <dd>ORD</dd>
-	        <dt>Created:</dt>
-	        <dd>December 2, 2014 @ 14:28</dd>
+            <section>
```
Comment (CITguy) [inline src/components/rxCollapse/docs/rxCollapse.html]: Do not use a nested `<section>`. Please place `rx-meta` elements directly within `rx-collapse`.

**Comment by 100stacks** [comment]
@CITguy thanks for the visual regression feedback https://github.com/rackerlabs/encore-ui/pull/1431#issuecomment-161378803

nested `<section>` removed. :+1:

**Comment by CITguy** [comment]
LGTM

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**